include/WCSimRootEvent.hh

```diff
@@ -27,6 +27,20 @@ class WCSimRootTrack : public TObject {
 
 private:
 
+  enum G4ProcessType {
+    hadElastic,
+    nCapture,
+    compt,
+    phot,
+    Cerenkov,
+    Scintillation,
+    eBrem,
+    annihil,
+    conv,
+    eIoni,
+    neutronInelastic
+  };
+
```

Comment:
I prefer use a different name to avoid conflicts with the geant4 built-iin `G4ProcessType`.
Also define this in `WCSimEnumerations.hh` instead, and implement `EnumAsString` for this enum.